### DISCUSSION TITLE
Can we provide an option to right-align the labels on a line chart?

### DISCUSSION THREAD

**Discussion by rayluo** [ChartsCSS/charts.css#47] [Ideas]
In charts.css, the line in line chart is consisted of many segments. Each segment has its own `start` and end (denoted by `size` in charts.css). [Their labels are currently displayed aligning to the center of each segment](https://chartscss.org/charts/line/#labels).

I assume many (if not the most) of the line charts are meant to emphasize the data as points, rather than the segment connecting those points. So, can we provide an option to right-align the labels to the right-hand side of each segment?

The developer could write their own css to customize the label alignment, but it seems not as straightforward as one might think. A global `.line th {align-items: end}` somehow does not work.

PS: Being the maintainer of the [Python binding Charts.css.py](https://github.com/rayluo/charts.css.py), I could generate that `align-items:end` as inline style for each `<th>` tag, but I figure it would be better if the upstream charts.css can provide a toggle class to enable this behavior out of the box.

**Comment by ramiy** [comment]
This feature already exist as a variable, check the [labels alignment](https://chartscss.org/components/labels/#align-labels) docs.

Also, see  [Charts.css cheatsheet](https://chartscss.org/development/cheat-sheet/) with all the available variables.

**Reply by rayluo** [reply]
Good to know. Somehow I did not notice that, partially because it does not seem to work, not even in the [official sample Align Labels](https://chartscss.org/components/labels/#align-labels). Is that a bug?

**Reply by ramiy** [reply]
Yes, it's a bug in version 0.9.0.

I already [fixed it](https://github.com/ChartsCSS/charts.css/blob/master/src/components/_labels.scss#L68-L82) in the repo, but it will be available only in the next release.

**Reply by rayluo** [reply]
No wonder.

Interestingly, that fix happened to be [the first commit immediately after the `0.9.0` release](https://github.com/ChartsCSS/charts.css/compare/0.9.0...master). And, the second last commit was a `1.0.0` commit, and the 4th last commit was about pie chart! Very exciting. :-D

So I guess the follow-up question would be "when will next release be released". :-)  It is understandable that you may want to take longer time to test 1.0.0 before its release. In such case, you may consider have a bugfix release 0.9.1 without pie chart in near future. Generally speaking, bugfix release should happen more timely and frequently than new feature release.
